docs/rock2/e20c/getting-started/quick-start.md

```diff
@@ -0,0 +1,227 @@
+---
+sidebar_position: 2
+---
+
+import { Section, Image } from "@site/src/utils/docs";
+
+# 快速上手
+
+## 实物图
+
+<img src="/img/radxa-e20c/radxa-e20c-machine.webp" width="500" alt="radxa-e20c pack" />
+
+要启动 Radxa E20C，您需要以下设备：
+
+- 一个 5V 的 DC 适配器，插头规格为 5.5x2.5mm，内为正极，外为地线。更多过供电详情，请参考[电源方案](./power-supply)。
```

Comment:
The Radxa E20C supports power from USB Type-C with 5V voltage, the suggested power source capability is 5V/3A with all the perpherials connected.